flink-table-store-e2e-tests/src/test/java/org/apache/flink/table/store/tests/HiveE2eTest.java

```diff
@@ -33,6 +34,7 @@
  * <p>NOTE: This test runs a complete Hadoop cluster in Docker, which requires a lot of memory. If
  * you're running this test locally, make sure that the memory limit of your Docker is at least 8GB.
  */
+@DisabledIfSystemProperty(named = "flink.version", matches = "1.14.*")
```

Comment:
Note: this class is not disabled if running it single from IDEA. See https://youtrack.jetbrains.com/issue/IDEA-194561